types/cyberblast__config/cyberblast__config-tests.ts

```diff
@@ -0,0 +1,5 @@
+import { Config } from "cyberblast__config";
```

Comment:
Please use the original name of the package. [What about scoped packages?](https://github.com/DefinitelyTyped/DefinitelyTyped/#what-about-scoped-packages)

```suggestion
import { Config } from "@cyberblast/config";
```